packages/kbn-custom-events/src/fullstory.js

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
I wonder if we should install fullstory as a package. From https://help.fullstory.com/hc/en-us/articles/360020828273-Getting-Started-with-FullStory, we should be able to install [`@fullstory/browser`](https://www.npmjs.com/package/@fullstory/browser). 

@elastic/kibana-operations what's the preference for this script?

I wonder if the NPM module would work because we are loading and initializing the module a bit late in the DOM due to our plugin architecture (not exactly following Fullstory's recommendation). Probably @joshdover can provide more context.

Reply:
We're loading this as lazily as possible to avoid loading 3rd party tracking code for customers not running on our Cloud environment. AFAICT in testing, it doesn't cause any issues and I confirmed with a support engineer from FullStory that this shouldn't be a problem.

Another reason we don't use the NPM package is that we actually require a security audit on this snippet from our internal InfoSec team before updating it. The version currently in the repo is the last one that has been validated.

Reply:
FYI This version contains no changes - I just moved it as is and changed the file type to `js`

Reply:
Makes sense! Thank you both!